### ISSUE TITLE
pak update from CRAN

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/pak#565]
```
Version: 0.7.0
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    progress.c:98:11: warning: too many arguments for format [-Wformat-extra-args]
    handle.c:115:20: warning: field precision specifier ‘.*’ expects argument of type ‘int’, but argument 3 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
    utils.c:87:24: warning: format ‘%d’ expects argument of type ‘int’, but argument 2 has type ‘long int’ [-Wformat=]
    create-time.c:147:13: warning: too many arguments for format [-Wformat-extra-args]
  See ‘/home/hornik/tmp/R.check/r-devel-gcc/Work/PKGS/pak.Rcheck/00install.out’ for details.
  * used C compiler: ‘gcc-13 (Debian 13.2.0-5) 13.2.0’
Flavor: [r-devel-linux-x86_64-debian-gcc](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/pak-00check.html)
```

**Comment by gaborcsardi** [comment]
Needs update from cli (https://github.com/r-lib/cli/issues/654), processx (https://github.com/r-lib/processx/issues/379) and curl (already updated on CRAN).

**Comment by gaborcsardi** [comment]
curl updated in 599429115d50aa76d044d0e10e0ad6c6ccb9112d

**Comment by gaborcsardi** [comment]
There is also 
```
  parse.c:38:32: warning: format not a string literal and no format arguments [-Wformat-security]
  push_parser.c:71:14: warning: format not a string literal and no format arguments [-Wformat-security]
  hbio.c:1629:5: warning: format not a string literal and no format arguments [-Wformat-security]
```
from jsonlite and lpSolve. jsonlite was already updated on CRAN.

**Comment by gaborcsardi** [comment]
jsonlite updated in 1c4327ed144d2728eab64cff9ac546a6ae24152f

**Comment by gaborcsardi** [comment]
Also needs lpSolve update (https://github.com/gaborcsardi/lpSolve/issues/19).

**Comment by gaborcsardi** [comment]
Fixed in #570.
